Sming/SmingCore/Network/MqttClient.cpp

```diff
@@ -46,22 +63,55 @@ void MqttClient::setKeepAlive(int seconds)
 void MqttClient::setPingRepeatTime(int seconds)
 {
 	if (PingRepeatTime > keepAlive)
-	   PingRepeatTime = keepAlive;
+		PingRepeatTime = keepAlive;
 	else
-	   PingRepeatTime = seconds;
+		PingRepeatTime = seconds;
 }
 
 bool MqttClient::setWill(const String& topic, const String& message, int QoS, bool retained /* = false*/)
 {
 	return mqtt_set_will(&broker, topic.c_str(), message.c_str(), QoS, retained);
 }
 
+bool MqttClient::connect(const URL& url, const String& clientName, uint32_t sslOptions)
+{
+	// eg						"mttqs://frank:frankspassword@frankserver:1883", "unique-client-name");
```

Comment:
Remove those comments from here. Add them to the header file using the document comment syntax.